lib/ui.js

```diff
@@ -145,6 +151,22 @@ module.exports = options => {
 	];
 
 	return printCommitLog(repositoryUrl)
-		.then(() => inquirer.prompt(prompts))
+		.then(hasCommits => {
+			if (!hasCommits) {
+				return inquirer.prompt([{
+					type: 'confirm',
+					name: 'confirm',
+					message: 'No commits found since previous release, continue?',
+					default: true
```

Comment:
Should it default to `true`?

Reply:
Wasn't sure about it either. It's easier as an end user if it's true, then you can just press enter but makes you think again if this is what you really want to do.

Reply:
Yeah, I don't feel strongly about either of them. I was just thinking that if you're publishing without any new commits, you're more likely to have done something wrong. But as long as we have a confirmation rather than just show a warning I'm ok with it :).

Reply:
Let's default it to `false`. 99% of the times you would get this message it's because you did something incorrect. Better to default to a safe value.

Reply:
Yes, you're right.